server/package.json

```diff
@@ -6,7 +6,10 @@
   "type": "module",
   "packageManager": "yarn@3.4.1",
   "scripts": {
-    "start-express": "node ./server.js"
+    "build": "ncc build ./src/server.ts -o dist -t --no-cache",
```

Comment:
Kopiert samme build som brukes i selvbetjening

Reply:
hva er ncc? Er dette en alias for noe?

Reply:
Ok, noe usikker om den er tilgjengelig på github og selvbetjening bygges på en litt annet måte.